src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DebateWorkbench.ts

```diff
@@ -53,7 +53,8 @@ export var documentDetailColumnDirective = (
 
 export var documentCreateColumnDirective = (
     adhConfig : AdhConfig.IService,
-    adhTopLevelState : AdhTopLevelState.Service
+    adhTopLevelState : AdhTopLevelState.Service,
+    adhResourceUrl
```

Comment:
Then it is required in as a module dependency in Module.ts, but it is not necessary to inject it here.